src/main/java/hudson/remoting/Util.java

```diff
@@ -60,6 +61,7 @@ public static void copy(InputStream in, OutputStream out) throws IOException {
     }
 
     @Nonnull
+    @SuppressFBWarnings(value = "PATH_TRAVERSAL_IN", justification = "This path exists within a temp directory so the potential traversal is limited.")
```

Comment:
Unless it's in some sort of jail, I don't see how being in a temporary directory makes it any less exploitable.

Reply:
It is in some weird kind of jail. I can't remember the details but I had trouble finding a way for this to escape. The things I tried didn't work.